test/pylib/manager_client.py

```diff
@@ -176,7 +176,8 @@ async def server_add(self, replace_cfg: Optional[ReplaceConfig] = None,
                          cmdline: Optional[List[str]] = None,
```

Comment:
Re commit message
```
The first one is that if we want to add a server while expecting
an error, we have to do it in two steps:
- call server_add with the start parameter set to False,
- call server_stop with the expected_error parameter.
```

I think you wanted `server_start` instead of `server_stop`.